### ISSUE TITLE
Is it possible to take an Spotify stream and use with the Superpowered SDK?

### ISSUE THREAD

**Issue by ManjitBedi** [spotify/ios-sdk#702]
I have got an existing app working with playing Spotify audio streams.  The next step is to see if it is possible to take the Spotify audio stream and apply effects using the Superpowered SDK.

http://superpowered.com

I am reading through the documentation.  

Has anyone tried to do something like this & how did you do it?

**Comment by d-kunin** [comment]
I am just afraid that what you want to do is not allowed by our Developer Terms of Use:

> 3. Spotify Content Restrictions. In addition to the foregoing, you must comply with the following restrictions related to Spotify Content
> **Editing, Modifying, and Altering.** Except as set out in this paragraph or agreed upon by Spotify in writing, you shall not edit, modify, alter or create derivative works of Spotify Content in any way. For example, you are not permitted to edit sound recordings accessible through the Spotify Platform to create preview clips; you may only use unmodified Audio Preview Clips available through our Spotify Platform. You may, however, adjust the size of metadata or cover art as necessary for a use authorized under this Agreement.

Please refer to https://developer.spotify.com/developer-terms-of-use/#section-iv-restrictions for more information.

**Comment by ManjitBedi** [comment]
Hello Dimitry,  I did see this mentioned elsewhere.  I was not certain that exceptions could be made if an agreement had been made with Spotify & individual companies or whatever. I don't deal with the legal stuff.  😊

I should mention the app is a for a recording artist who has been in discussion with Spotify about his music etc;  I am not part of that discussion - just a humble developer.  He wants this feature for his own music when streamed on Spotify.  I will communicate that back to him.   I don't know if he has gone over this particular details of the app with your colleagues.

In short, this is not a technical issue at the moment.
